src/ServicePulse.Host/vue/src/composables/isRouteSelected.ts

```diff
@@ -0,0 +1,15 @@
+import { useLink, useRoute } from "vue-router";
+
+export default function isRouteSelected(path: string) {
+  const route = useRoute();
+
+  if (route.path === path) {
+    return true;
+  }
+
+  if (`${path}/` === route.path) {
+    return true;
+  }
+
+  return useLink({ to: path }).route.value.name === route.name;
```

Comment:
This one handles grandchildren routes, when we want to still select the correct tab, but the route is a child of the parent route.